integrations/aspnetcore/src/Scalar.AspNetCore/Options/ScalarOptions.cs

```diff
@@ -7,8 +8,17 @@ namespace Scalar.AspNetCore;
 /// Represents all available options for the Scalar API reference.
 /// Based on <a href="https://github.com/scalar/scalar/blob/main/documentation/configuration.md">Configuration</a>.
 /// </summary>
-public sealed class ScalarOptions
+public sealed partial class ScalarOptions
 {
+
+    [GeneratedRegex("^[a-zA-Z]+://.*", RegexOptions.IgnoreCase | RegexOptions.Compiled)]
```

Comment:
The `RegexOptions.Compiled` flag will be ignored by the source generator and therefore can be removed.
```suggestion
    [GeneratedRegex("^[a-zA-Z]+://.*", RegexOptions.IgnoreCase)]
```